### DISCUSSION TITLE
Logo size limitation missing

### DISCUSSION THREAD

**Discussion by obrunsmann** [laravel/nova-issues#3888] [General]
- Laravel Version: v8.83.7
- Nova Version: 4.0.5
- PHP Version: 8.1.4
- Browser type and version: Chrome latest

I just added our logo to the new brand config but it seems like there is a reasonable size missing. Maybe giving the embedded SVG a max height of the nav bar?

<img width="1536" alt="Bildschirmfoto 2022-04-07 um 19 02 15" src="https://user-images.githubusercontent.com/763812/162257929-e81285d0-10da-4815-8b71-893fbd7d40ec.png">

**Comment by davidhemphill** [comment]
You can hardcode the size inside your logo SVG file.

**Reply by obrunsmann** [reply]
But there is actually no reason not to simply set the height limit in the template? We use the logo in other places as well. Before editing the SVG file, it is much easier to limit the container?

**Reply by michaelhume** [reply]
Hardcoding also leads to issues as you end up with a small version of the logo on the login screen.

**Reply by obrunsmann** [reply]
That's right, I didn't even think of that. I think it should happen directly in context. That's what SVGs are just perfect for.

**Reply by davidhemphill** [reply]
Ok guys, I just pushed an update to Nova that adds a height class to authentication and main layouts. 👍
